backend/app/tests/api/routes/test_responses.py

```
@@ -120,3 +120,66 @@ def test_responses_endpoint_without_vector_store(
+
+
+@patch("app.api.routes.responses.OpenAI")
+@patch("app.api.routes.responses.get_provider_credential")
+@patch("app.api.routes.responses.get_assistant_by_id")
+@patch("app.api.routes.responses.create_openai_conversation")
+def test_responses_endpoint_stores_conversation(
+    mock_create_conversation,
+    mock_get_assistant,
+    mock_get_credential,
+    mock_openai,
+    db,
```

Ruff (0.12.2)

Unused function argument: `db`

(ARG001)